packages/blade/src/utils/makeAccessible/makeAccessible.web.ts

```diff
@@ -0,0 +1,25 @@
+import { accessibilityMap } from './accessibilityMapWeb';
+import type { AccessibilityMap, AccessibilityProps } from './makeAccessible.d';
+
+const makeAccessible = (props: Partial<AccessibilityProps>): Record<string, unknown> => {
+  const newProps: Record<string, unknown> = {};
```

Comment:
why is it `Record<string, unknown>`? the new props shall have `key:value` pairs which are valid right?

Reply:
It is defining key:value pairs. just that we don't know what the value would be. It can be either boolean, string, 'mixed' or anything. I should change this to `any` actually.